src/app/components/Login/Login.js

```diff
@@ -62,7 +62,7 @@ export class Login extends Component {
         // getTransactions(networks, selectedNetworkId, account).then(transactions => setTransactions(transactions))
         Neoscan.setNet(selectedNetworkId)
         Neoscan.getBalance(account.address).then(results => setBalance(results.neo, results.gas))
-        Neoscan.getTxsByAddress(account.address).then(results => setTransactions(results))
+        Neoscan.getTxsByAddressUrl(account.address).then(results => setTransactions(results))
```

Comment:
I think this is a bug.

Reply:
The method doesn't exist in the Neoscan file :P 
Also got a compilation error pointing this out. I also did a search and wasn't able to find the function

Reply:
Weird. I'll check into it.

Reply:
keep me posted! 🎉

Reply:
Alright. So I can remove this line and then we're done for this PR?